src/Puny.php

```diff
@@ -65,12 +67,15 @@ public function run(string $path)
 
         self::requireIfReadable($this->root.'/tidy-up.php');
 
+        $end = microtime(true);
+
         Console::write(sprintf(
-            "%s tests run. %s passed. %s failed. %s skipped.",
+            "%s tests run. %s passed. %s failed. %s skipped. %02.2f ms",
             count(static::$tests),
             count(static::$tests) - $this->failed - $this->skipped,
             $this->failed,
             $this->skipped,
+            $end - $start
```

Comment:
`$end` is only being used once so I think we can change this to:
```suggestion
            microtime(true) - $start
```